.gitmodules

```diff
@@ -29,3 +29,6 @@
     path = external/Xamarin.MacDev
     url = ../../xamarin/Xamarin.MacDev
     branch = master
+[submodule "external/macios-binaries"]
+	path = external/macios-binaries
+	url = https://github.com/xamarin/macios-binaries
```

Comment:
Use a relative url (look at the others) instead.

Also indentation is different than the others (whitespace issue)?

Reply:
Strange, it is what git submodule add did to the file. I did not manually change it.

Let me fix this up.